core/freq_params_traffic/concurrency_stat_slot.go

```diff
@@ -0,0 +1,70 @@
+package freq_params_traffic
+
+import (
+	"github.com/alibaba/sentinel-golang/core/base"
+	"github.com/alibaba/sentinel-golang/util"
+)
+
+type ConcurrencyStatSlot struct {
+}
+
+func (c ConcurrencyStatSlot) OnEntryPassed(ctx *base.EntryContext) {
+	res := ctx.Resource.Name()
+	args := ctx.Input.Args
+	tcs := getTrafficControllerListFor(res)
+	for _, tc := range tcs {
+		if tc == nil {
+			logger.Warnf("Nil traffic controller was found, res: %s", res)
+			continue
+		}
+		idx := tc.getParamIndex()
+		if idx < 0 {
+			logger.Warnf("The TrafficShapingController's paramIndex(%d) less than 0.", idx)
+		}
+		if len(args) <= idx {
+			logger.Warnf("The index of rule is not existed in args. The TrafficShapingController's paramIndex is %d, args: %+v, tc: %+v", idx, args, tc)
```

Comment:
Ditto for logging level. Also typo: is not existed -> does not exist

Reply:
Okay